com.unity.renderstreaming/Runtime/Scripts/Signaling/HttpSignaling.cs

```diff
@@ -24,8 +22,12 @@ public class HttpSignaling : ISignaling
 
         public HttpSignaling(SignalingSettings signalingSettings, SynchronizationContext mainThreadContext)
         {
-            m_url = signalingSettings.url;
-            m_timeout = ((HttpSignalingSettings) signalingSettings).interval;
+            if (signalingSettings == null)
+                throw new ArgumentNullException(nameof(signalingSettings));
+            if (signalingSettings is not HttpSignalingSettings settings)
```

Comment:
In Unity 2020.3 compile error happened, ` is not` syntax does not supported

Reply:
fixed it